shard-operator/src/main/java/com/redhat/service/bridge/shard/operator/monitoring/ServiceMonitorServiceImpl.java

```diff
@@ -31,35 +31,31 @@
     @Override
     public Optional<ServiceMonitor> fetchOrCreateServiceMonitor(final CustomResource resource, final Service service, final String componentName) {
         if (!ServiceMonitorClient.isServiceMonitorAvailable(kubernetesClient)) {
-            LOGGER.debug("Prometheus Operator is not available in this cluster");
+            LOGGER.warn("Prometheus Operator is not available in this cluster");
```

Comment:
Please revert to DEBUG since the controllers are already warning a similar message.